Tests are broken right now

Master as of 568d6a64416bd1b89f44a54b8459229b356e2aba has some breaking unittests. 

```
# gradle check
:compileJava UP-TO-DATE
:processResources UP-TO-DATE
:classes UP-TO-DATE
:compileTestJava UP-TO-DATE
:processTestResources
:testClasses
:test
objc[94315]: Class JavaLaunchHelper is implemented in both /Library/Java/JavaVirtualMachines/jdk1.8.0_66.jdk/Contents/Home/bin/java and /Library/Java/JavaVirtualMachines/jdk1.8.0_66.jdk/Contents/Home/jre/lib/libinstrument.dylib. One of the two will be used. Which one is undefined.

io.github.swagger2markup.AsciidocConverterTest > testSwagger2AsciiDocConversionWithPolymorphismAsIsOrdering FAILED
    org.junit.ComparisonFailure at AsciidocConverterTest.java:527

io.github.swagger2markup.AsciidocConverterTest > testSwagger2AsciiDocConversion FAILED
    org.junit.ComparisonFailure at AsciidocConverterTest.java:80

io.github.swagger2markup.AsciidocConverterTest > testSwagger2AsciiDocGroupedByTags FAILED
    org.junit.ComparisonFailure at AsciidocConverterTest.java:257

io.github.swagger2markup.AsciidocConverterTest > testSwagger2AsciiDocConversionWithResponseHeaders FAILED
    org.junit.ComparisonFailure at AsciidocConverterTest.java:550

io.github.swagger2markup.AsciidocConverterTest > testSwagger2AsciiDocConversionWithInterDocumentCrossReferences FAILED
    org.junit.ComparisonFailure at AsciidocConverterTest.java:122

io.github.swagger2markup.AsciidocConverterTest > testSwagger2AsciiDocConversionWithPolymorphism FAILED
    org.junit.ComparisonFailure at AsciidocConverterTest.java:503

io.github.swagger2markup.AsciidocConverterTest > testSwagger2AsciiDocConversionWithGeneratedExamples FAILED
    org.junit.ComparisonFailure at AsciidocConverterTest.java:190

io.github.swagger2markup.AsciidocConverterTest > testSwagger2AsciiDocConversionFromString FAILED
    org.junit.ComparisonFailure at AsciidocConverterTest.java:141

io.github.swagger2markup.AsciidocConverterTest > testSwagger2AsciiDocConversionInstagram FAILED
    org.junit.ComparisonFailure at AsciidocConverterTest.java:99

io.github.swagger2markup.AsciidocConverterTest > testSwagger2AsciiDocConversionWithExamples FAILED
    org.junit.ComparisonFailure at AsciidocConverterTest.java:165

io.github.swagger2markup.GeneralConverterTest > testToFileWithoutExtension FAILED
    org.junit.ComparisonFailure at GeneralConverterTest.java:65

io.github.swagger2markup.MarkdownConverterTest > testSwagger2MarkdownConversionWithInterDocumentCrossReferences FAILED
    org.junit.ComparisonFailure at MarkdownConverterTest.java:102

io.github.swagger2markup.MarkdownConverterTest > testSwagger2MarkdownConversionWithResponseHeaders FAILED
    org.junit.ComparisonFailure at MarkdownConverterTest.java:154

io.github.swagger2markup.MarkdownConverterTest > testSwagger2MarkdownConversion FAILED
    org.junit.ComparisonFailure at MarkdownConverterTest.java:78

39 tests completed, 14 failed
:test FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///Users/zerok/code/swagger2markup/build/reports/tests/index.html

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 1 mins 24.642 secs
```
The issue seems to be related with outdated "expected" files with regards to default/minimum/maximum/etc. renderings. 

I tried to bisect the errors but didn't get that far. In some cases the default/minimum/maximum properties of definitions are rendered differently, in others the order of elements is reversed. I'm not even sure that this is an issue within swagger2markup itself or one of the swagger.io dependencies. Only that the expected output is probably no longer valid.

Sadly, I'm completely new to the code-base, so I'm most likely simply looking at the wrong places right now 😃 

Please note that the tests pass in v1.0.0 if I set the io.github.swagger2markup:markup-document-builder dependency to 1.0.0 (instead of the 1.0.0-SNAPSHOT). With the snapshot building already fails.

--- a/src/test/resources/expected/asciidoc/instagram/overview.adoc
+++ b/src/test/resources/expected/asciidoc/instagram/overview.adoc
@@ -18,9 +18,11 @@ All endpoints are only accessible via https and are located at
 `api.instagram.com`. For instance: you can grab the most popular photos at
 the moment by accessing the following URL with your client ID
 (replace CLIENT-ID with your own):
-`
+
+----
   https://api.instagram.com/v1/media/popular?client_id=CLIENT-ID
-`
+----
+
 You're best off using an access_token for the authenticated user for each
 endpoint, though many endpoints don't require it.
 In some cases an access_token will give you more access to information, and

